src/app/shared/services/mock.services.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable } from '@angular/core';
```

Comment:
Nice, but i wonder if we could instead setup a real server to mock these data, so that we can simply replace the url of the graphQL api between dev and prod.
The skeleton has already an express server running at :3000, but it's currently configured to return data via REST, i think, ideally we could see if we can replace this conf or the entire server, i created an example here: https://github.com/planctus/decoupled-frontend/pull/2 of a conf for a graphQL client and an example of the way to perform the query.

Reply:
I improved the data mockups but as discussed in meeting we will keep it like this for now.